### PR TITLE
Azure Security Center- fix subscription id issues

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fixed an issue where commands which do not require a `subscription_id` parameter failed for the lack of it.
Docs improvements

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/README.md [hunk 1]
```diff
@@ -1,186 +1,107 @@
...
+    - **Token (received from the admin consent - see Detailed Instructions (?) section)**
+    - **Key (received from the admin consent - see Detailed Instructions (?)**
+    - **Trust any certificate (not secure)**
+    - **Use system proxy settings**
+    - **Default subscription ID to use**
+4. Click **Test** to validate the new instance.
+
+
+## Commands
+#### Subscription ID
```
Comment (Bargenish) [inline Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/README.md:46]: @Itay4 Ive added a subscription id section here and added a comment for every command that requires a subscription id.

**Review by Itay4 — APPROVED** [review]
looks great
please add to the docs the list of commands that require the subscription id

**Comment by Bargenish** [comment]
@Itay4 in additional to the comment in each command or instead?

**Comment by Itay4** [comment]
> @Itay4 in additional to the comment in each command or instead?

sorry i missed that
lets change to `Require Subscription ID`

**Comment by Bargenish** [comment]
@kirbles19 although it seems a major change was made to the README file most of it its the HTML to markdown conversion. The relevant changes are in lines: 46-49
and the release notes of course.
Thanks 🙂

**Comment by kirbles19** [comment]
@Bargenish Done.

**Pull request merged**